static/js/components/EmploymentForm.js

```diff
@@ -16,6 +16,14 @@ import ProfileFormFields from '../util/ProfileFormFields';
 import ConfirmDeletion from './ConfirmDeletion';
 
 class EmploymentForm extends ProfileFormFields {
+  static propTypes = {
```

Comment:
I did a `console.log` and it looks like this replaces the propTypes so you'll need to combine the two or move it into the parent class like you said